modules/logging-release-notes-5-8-19.adoc

```diff
@@ -0,0 +1,25 @@
+// module included in /logging/logging-5-8-release-notes
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-5-8-19_{context}"]
+= Logging 5.8.19
+
+This release includes link:https://access.redhat.com/errata/RHBA-2025:3447[RHBA-2025:3447] and link:https://access.redhat.com/errata/RHSA-2025:3448[RHSA-2025:3448].
+
+[id="logging-release-notes-5-8-19-cves_{context}"]
+== CVEs
+
+* link:https://access.redhat.com/security/cve/CVE-2019-12900[CVE-2019-12900]
+* link:https://access.redhat.com/security/cve/CVE-2020-11023[CVE-2020-11023]
+* link:https://access.redhat.com/security/cve/CVE-2024-2236[CVE-2024-2236]
+* link:https://access.redhat.com/security/cve/CVE-2024-2511[CVE-2024-2511]
+* link:https://access.redhat.com/security/cve/CVE-2024-3596[CVE-2024-3596]
+* link:https://access.redhat.com/security/cve/CVE-2024-4603[CVE-2024-4603]
+* link:https://access.redhat.com/security/cve/CVE-2024-4741[CVE-2024-4741]
+* link:https://access.redhat.com/security/cve/CVE-2024-5535[CVE-2024-5535]
+* link:https://access.redhat.com/security/cve/CVE-2024-12797[CVE-2024-12797]
+* link:https://access.redhat.com/security/cve/CVE-2024-56171[CVE-2024-12797]
+* link:https://access.redhat.com/security/cve/CVE-2025-1094[CVE-2025-1094]
+* link:https://access.redhat.com/security/cve/CVE-2025-1244[CVE-2025-1244]
+* link:https://access.redhat.com/security/cve/CVE-2025-24855[CVE-2025-24855]
```

Comment:
I can't see these three (CVE-2025-1094, CVE-2025-1244, CVE-2025-24855) in the advisories.

Reply:
removed